src/components/Sidebar/CalendarPanel.vue

```diff
@@ -1,22 +1,42 @@
 <template>
-  <SidebarPanel :isInSidebar="isInSidebar" header="Calendar" headerIcon="@/assets/calendar.svg">
-    <b-calendar locale="en-US" style="width:200px"></b-calendar>
-  </SidebarPanel>
+  <DatePicker mode="range" v-model="dates" />
 </template>
 
 <script lang="ts">
-  import { Component, Prop, Vue } from 'vue-property-decorator';
+  import { Component, Prop, Vue, Watch } from 'vue-property-decorator';
   import { mixins } from 'vue-class-component';
   import SidebarPanel from '@/components/Sidebar/SidebarPanel.vue';
   import SidebarPanelMixin from '@/mixins/SidebarPanel';
+  import DatePicker from 'v-calendar/lib/components/date-picker.umd';
 
   @Component({
     components: {
       SidebarPanel,
+      DatePicker,
     },
   })
   export default class CalendarPanel extends mixins(SidebarPanelMixin) {
-    @Prop() tags!: string[];
+    private dates = {};
+
+    mounted() {
+      const { from_created, to_created } = this.$route.query;
+
+      if (from_created && to_created)
+        this.dates = {
+          start: new Date(+from_created),
```

Comment:
I am not a fan of using the epoch number in the query.
It seems like the Date constructor accepts YYYY-MM-DD; I think this will be better, as it also allows users to edit the date in the url.

Reply:
good idea, I feel the same way. I can make this change as part of a separate PR.